assets/blueprints/radiswap/src/lib.rs

```diff
@@ -69,6 +69,9 @@ mod radiswap {
 
             let output_amount = (input_amount * output_reserves) / (input_reserves + input_amount);
 
+            // NOTE: It's the responsibility of the user of the pool to do the appropriate rounding
```

Comment:
There is nothing in the pool to fix for this one. `protected_withdraw` fails if the amount withdrawn is invalid for the resource. As an example, if we attempt to withdraw `1.1111` for a resource with a divisibility of 2 then this method fails. This method should not do any silent rounding as this might not be the intended logic that the owner of the pool wants. All other methods take the divisibility into account when doing their calculations. 

A doc comment is nice, but I think we mainly need to highlight this in our docs.